ui/apps/platform/eslint-plugins/accessibilityPlugin.js

```diff
@@ -0,0 +1,43 @@
+const accessibilityPlugin = {
+    meta: {
+        name: 'accessibilityPlugin',
+        version: '0.0.1',
+    },
+    rules: {
+        'require-Alert-component': {
+            // Require alternative markup to prevent axe DevTools issue:
+            // Heading levels should only increase by one
+            meta: {
+                type: 'problem',
+                docs: {
+                    description: 'Require that Alert element has component="p" prop',
+                },
+                fixable: 'code',
+                schema: [],
+            },
+            create(context) {
+                return {
+                    JSXOpeningElement(node) {
+                        if (node.name?.name === 'Alert') {
+                            if (
+                                !node.attributes.some((nodeAttribute) => {
+                                    return (
+                                        nodeAttribute.name?.name === 'component' &&
+                                        nodeAttribute.value?.value === 'p'
+                                    );
+                                })
+                            ) {
+                                context.report({
+                                    node,
+                                    message: 'Alert element requires component="p" prop',
```

Comment:
What do you think about:
```suggestion
                                    message: 'Alert element requires component="p" prop to prevent invalid accessible heading levels on page',
```

Reply:
Thank you for suggesting what **and why**.

The only reason I might hesitate is 102 characters wide with the original text:

```
src/Containers/AccessControl/AccessScopes/AccessScopes.tsx
  68:21  error  Alert element requires component="p" prop  accessibilityPlugin/require-Alert-component
  92:21  error  Alert element requires component="p" prop  accessibilityPlugin/require-Alert-component
```

As an afterthought, maybe influenced by former tour of duty on ESLint rule docs:
https://eslint.org/docs/latest/rules/